consent/manager_sql.go

```diff
@@ -287,7 +287,11 @@ func (m *SQLManager) HandleConsentRequest(ctx context.Context, challenge string,
 		strings.Join(sqlParamsConsentRequestHandled, ", "),
 		":"+strings.Join(sqlParamsConsentRequestHandled, ", :"),
 	), d); err != nil {
-		return nil, sqlcon.HandleError(err)
+		err = sqlcon.HandleError(err)
+		if errors.Cause(err) == sqlcon.ErrUniqueViolation {
```

Comment:
In this case we wouldn't override the existing value. However, I think that's what we should do here as long as `was_used` is false.